components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -129,230 +130,238 @@ + (NSDictionary *)catalogMetadata {
   };
 }
 
+#pragma mark - UIViewController
+
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  self.contentView.frame = self.view.bounds;
+  self.contentViewLabel.center = self.contentView.center;
+  CGRect exampleListTableViewFrame =
+      CGRectMake(0, self.view.bounds.size.height - exampleListTableViewHeight,
+                 self.view.bounds.size.width, exampleListTableViewHeight);
+  self.exampleListTableView.frame = exampleListTableViewFrame;
+
+  CGSize bannerViewSize =
+      [self.bannerView sizeThatFits:CGSizeMake(CGRectGetWidth(self.view.bounds), CGFLOAT_MAX)];
+
+  // Adjust bannerViewContainer's frame
+  CGFloat topAreaInset = 0.0f;
+  if (@available(iOS 11.0, *)) {
+    topAreaInset = self.view.safeAreaInsets.top;
+  }
+  self.bannerView.frame =
+      CGRectMake(0.0f, topAreaInset, bannerViewSize.width, bannerViewSize.height);
```

Comment:
You should probably constrain the banner view's width to be no greater than the view's bounds.  `sizeThatFits:` does not need to be constrained to the passed size.